### PR TITLE
Report on pantheon desktop work

### PR DESCRIPTION
**Pull request by OlivierDuchateau**
My report for the pantheon desktop

### CODE REVIEW

File: 2022q2/pantheon.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Links: +
+link:https://elementary.io/[elementary OS] URL: link:https://elementary.io/[https://elementary.io] +
+link:https://codeberg.org/olivierd/freebsd-ports-elementary[Development repository] URL: link:https://codeberg.org/olivierd/freebsd-ports-elementary[https://codeberg.org/olivierd/freebsd-ports-elementary] +
+
+Contact: Olivier Duchateau <duchateau.olivier@gmail.com>
+
+The Pantheon desktop environment is designed for elementary OS. It builds on GNOME technologies (such Mutter, GNOME Shell, GTK 3 and 4) and it is written with Vala language.
+
+The goal is to have new desktop for users. Some features are not well supported, but we can have full session.
```
Comment (pauamma) [inline 2022q2/pantheon.adoc:11]: "have a new desktop"
"can have a full session" (also, not sure what you mean by "full session" - can you clarify?)

File: 2022q2/pantheon.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+I have submitted several patches, especially:
+
+* `x11-toolkits/granite7`
+* `devel/libgee` update to 0.20.5 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=262893[bug #262893]
+* `sysutils/bamf` update to 0.5.6 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264203[bug #264203]
+
+==== Open tasks
+
+* Add support of user settings (it is heavily targetted for Ubuntu)
+* Finish to port wingpanel-indicator-power (power management)
```
Comment (pauamma) [inline 2022q2/pantheon.adoc:24]: "Finish porting"

File: 2022q2/pantheon.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+I have submitted several patches, especially:
+
+* `x11-toolkits/granite7`
+* `devel/libgee` update to 0.20.5 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=262893[bug #262893]
+* `sysutils/bamf` update to 0.5.6 link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=264203[bug #264203]
+
+==== Open tasks
+
+* Add support of user settings (it is heavily targetted for Ubuntu)
```
Comment (pauamma) [inline 2022q2/pantheon.adoc:23]: Maybe "very Ubuntu-centric" if you mean it expects to run in an Ubuntu environment and relies on it both for its UI and  for the settings it let you control?

File: 2022q2/pantheon.adoc (hunk 1 above)
Comment (OlivierDuchateau) [reply 2022q2/pantheon.adoc]: Even if desktop environment is not supported by upstream, it is daily usable.

File: 2022q2/pantheon.adoc [hunk 4]
```diff
@@ -6,9 +6,9 @@ link:https://codeberg.org/olivierd/freebsd-ports-elementary[Development reposito
 
 Contact: Olivier Duchateau <duchateau.olivier@gmail.com>
 
-The Pantheon desktop environment is designed for elementary OS. It builds on GNOME technologies (such Mutter, GNOME Shell, GTK 3 and 4) and it is written with Vala language.
+The Pantheon desktop environment is designed for elementary OS. It builds on GNOME technologies (such as Mutter, GNOME Shell, GTK 3 and 4) and it is written in Vala.
 
-The goal is to have new desktop for users. Some features are not well supported, but we can have full session.
+The goal is to have a new desktop for users. Some features are not well supported, but we can have full session.
```
Comment (pauamma) [inline 2022q2/pantheon.adoc:11]: "a full session", but that's minor, so I can fix it after merging.

**Review by pauamma — APPROVED** [review]

**Pull request merged**